Example/Backpack/SwiftUI/Components/TextField/TextFieldExampleView.swift

```diff
@@ -0,0 +1,61 @@
+//
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright © 2023 Skyscanner Ltd. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import SwiftUI
+import Backpack_SwiftUI
+
+struct TextFieldExampleView: View {
+    @State var text = "Value"
+    @State var validationFieldText = ""
+    
+    var body: some View {
+        ScrollView {
+            VStack {
+                BPKTextField(.constant(""))
```

Comment:
Having a constant like this will 'break' the example, as a consumer is unable to input anything and the textfield 'blinks' when trying to force input in.

can we consider another @State for this, so it doesn't 'feel broken'?

Reply:
yeap